pkg/cli/filesystem/osfs.go

```diff
@@ -0,0 +1,41 @@
+package filesystem
+
+import (
+	"io/fs"
+	"os"
+)
+
+// OSFileSystem is an implementation of the FileSystem interface that uses the OS filesystem.
+// It uses the methods from the os package to interact with the filesystem.
+type OSFileSystem struct{}
+
+var _ FileSystem = (*OSFileSystem)(nil)
+
+func NewOSFS() *OSFileSystem {
+	return &OSFileSystem{}
+}
+
+func (osfs OSFileSystem) Create(name string) (fs.File, error) {
```

Comment:
Should we have unit testing for this abstraction? There is testing for`MemMapFS`

Reply:
These are just a wrapper over the os functions so I don't think unit tests are necessary.